src/http/request.rs

```diff
@@ -201,7 +201,7 @@ where
             } else {
                 let mut header_parts = header_line.splitn(2, ':');
                 if let (Some(header_key),Some(header_val)) = (header_parts.next(),header_parts.next()) {
-                    headers_out.insert(header_key.trim().to_string(),header_val.trim().to_string());
+                    headers_out.insert(header_key.trim().to_string().into(),header_val.trim().to_string());
```

Comment:
any way to avoid the into?

Reply:
Because it's too long?
Remove to_string

Reply:
😂, because I‘d like to modify code as less as possible.
I'm wondering any `implicit coercion`, which may be bad practice.

This code seems feasible for me now,
Do you have any other suggestions?

Reply:
😅
very great 👍.
thank you.
I will be back in just 3 days.
I am on my way to a place where there is no internet.
If you find something yourself, do it and we will apply it

Reply:
nice~~
enjoy yourself.